packages/wonder-blocks-testing/src/fixtures/__tests__/combine-options.test.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+import * as CombineModule from "../combine.js";
+
+import {combineOptions} from "../combine-options.js";
+
+jest.mock("../combine.js");
+
+describe("#combineOptions", () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it("should call combine once per property per object", () => {
+        // Arrange
+        const toBeCombined = [
+            {a: "test1", b: "test1"},
+            {b: "test2"},
+            {a: "test3", c: "test3"},
+        ];
+        const combineSpy = jest.spyOn(CombineModule, "combine");
+
+        // Act
+        combineOptions(...toBeCombined);
+
+        // Assert
+        expect(combineSpy).toHaveBeenCalledTimes(5);
+    });
+
+    it("should ignore falsy args", () => {
+        // Arrange
+        const toBeCombined = [null, {a: "test"}, {b: "test"}, 0, undefined];
+        const combineSpy = jest.spyOn(CombineModule, "combine");
+
+        // Act
+        combineOptions(...toBeCombined);
+
+        // Assert
+        expect(combineSpy).toHaveBeenCalledTimes(2);
+    });
+
+    it("should return the combined object", () => {
+        // Arrange
+        const toBeCombined = [
+            {a: "test1", b: "test1"},
+            {b: "test2"},
+            {a: "test3", c: "test3"},
+        ];
+        jest.spyOn(CombineModule, "combine").mockImplementation(
+            // Just for testing, we know the values are strings, so let's
+            // combine them with concatenation so we see the order of
+            // combination in the result.
+            (v1, v2) => `${v1 === undefined ? "" : v1}${v2}`,
+        );
```

Comment:
Can't the order be inferred by which values end up in the result?

Reply:
Yes, that's what the comment says. Can you clarify what you mean? Is there a change you're suggesting?

Reply:
If we didn't have this mock implementation here, the result would be:
```
{
  a: "test3",
  b: "test2",
  c: "test3",
}
```
We could assert that result.  This mock doesn't seem like it's adding much.  Apologies if I misunderstanding what this test is testing.

Reply:
I can see how you may reach that conclusion, but it won't work like that. The jest.mock call at the top of the file ensures we isolate the code so the original implementation won't be available here. I want to assert the code runs as-is without relying that other code works correctly.

Reply:
No worries. It's easy to do. Would be great if there were some sort of IDE highlighting for mocked calls to make it clearer.